### PR TITLE
feat(onboarding): added new tokens

### PR DESCRIPTION
**Pull request by mohd-kashif**
TICKET: COIN-1870

### CODE REVIEW

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -4089,6 +4089,14 @@ export const coins = CoinMap.fromCoins([
     '0xeb40045347531f06179ffb3c6be33d5b35a52ebd',
     UnderlyingAsset.GHUB
   ),
+  erc20(
+    'd3c6a9b2-1551-401c-b576-db429d48b895',
+    'ghst',
+    'Aavegotchi',
+    18,
+    '0x3F382DbD960E3a9bbCeaE22651E88158d2791550',
```
Comment (MohammedRyaan786) [inline modules/statics/src/coins.ts:4097]: please make sure all letters are in lowercase

File: modules/statics/src/coins.ts [hunk 2]
```diff
@@ -11990,6 +12054,30 @@ export const coins = CoinMap.fromCoins([
...
+    18,
+    '0x72fF5742319eF07061836F5C924aC6D72c919080',
+    UnderlyingAsset['bsc:gft']
+  ),
+  bscToken(
+    '71861b2e-03b0-4101-81ea-ea35b13d48d1',
+    'bsc:glmr',
+    'Moonbeam',
+    18,
+    '0x76F3CE6aF26de7A9854dBD153ACd8f46a2Cf5133',
```
Comment (MohammedRyaan786) [inline modules/statics/src/coins.ts:12070]: lower case please

File: modules/statics/src/coins.ts [hunk 3]
```diff
@@ -11990,6 +12054,30 @@ export const coins = CoinMap.fromCoins([
...
+    UnderlyingAsset['bsc:glmr']
+  ),
+  bscToken(
+    'bcc8389e-a3bf-493d-89a8-3d95676e817c',
+    'bsc:gmt',
+    'Green Metaverse Token',
+    18,
+    '0x3019BF2a2eF8040C242C9a4c5c4BD4C81678b2A1',
```
Comment (MohammedRyaan786) [inline modules/statics/src/coins.ts:12078]: lower case please

File: modules/statics/src/coins.ts [hunk 4]
```diff
@@ -11990,6 +12054,30 @@ export const coins = CoinMap.fromCoins([
...
+    'Moonbeam',
+    18,
+    '0x76F3CE6aF26de7A9854dBD153ACd8f46a2Cf5133',
+    UnderlyingAsset['bsc:glmr']
+  ),
+  bscToken(
+    'bcc8389e-a3bf-493d-89a8-3d95676e817c',
+    'bsc:gmt',
+    'Green Metaverse Token',
+    18,
```
Comment (MohammedRyaan786) [inline modules/statics/src/coins.ts:12077]: decimal value is 8 as per explorer: [0x3019BF2a2eF8040C242C9a4c5c4BD4C81678b2A1](https://bscscan.com/token/0x3019BF2a2eF8040C242C9a4c5c4BD4C81678b2A1)

File: modules/statics/src/coins.ts (hunk 1 above)
Comment (mohd-kashif) [reply modules/statics/src/coins.ts]: done

File: modules/statics/src/coins.ts (hunk 2 above)
Comment (mohd-kashif) [reply modules/statics/src/coins.ts]: done

File: modules/statics/src/coins.ts (hunk 3 above)
Comment (mohd-kashif) [reply modules/statics/src/coins.ts]: done

File: modules/statics/src/coins.ts (hunk 4 above)
Comment (mohd-kashif) [reply modules/statics/src/coins.ts]: done

File: modules/statics/src/coins.ts [hunk 5]
```diff
@@ -4241,6 +4249,22 @@ export const coins = CoinMap.fromCoins([
...
+    'Highstreet',
+    6,
+    '0x71Ab77b7dbB4fa7e017BC15090b2163221420282',
+    UnderlyingAsset.HIGH
+  ),
+  erc20(
+    '713f7531-d5ba-4c38-9c21-64458cb1bcf9',
+    'hifi',
+    'Hifi Finance',
+    6,
```
[the remark this answers is not in the document]
Comment (mohd-kashif) [reply modules/statics/src/coins.ts]: done

**Pull request closed without merging**